### DISCUSSION TITLE
Windows 10 Install

### DISCUSSION THREAD

**Discussion by GoatedAlpaca** [seerr-team/seerr#938] [Q&A]
Hey, are you still active here?

I'm trying to install Jellyseer on windows and running into some issues when doing Yarn Build and yarn start.

When I'm doing Yarn Start I get this error:

$ next build (node:2240) [DEP0040] DeprecationWarning: The punycode module is deprecated. Please use a userland alternative instead. (Use node --trace-deprecation ... to show where the warning was created)

I've on the latest nodeJS, Cypress and Yarn. And can't seen to figure out what the issue is. I tried running it with Node v22.6.0 and node-v20.16.0LTS but both get that issue.

**Comment by fallenbagel** [comment]
We carry out support on discord. Please hop onto discord.
Also latest jellyseerr doesn't support latest nodejs. It's in the docs that we only support v18.
